### ISSUE TITLE
NTR: Adipic acid high in urine (Lab: chemistry: organic acids)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8599]
New term request Adipic acid high in urine (segal_030513144143): Dicarboxylic acid, adipate; hexanedioic  UMLS: 4315767

**Comment by pnrobinson** [comment]
@MickeySegal
I suspect that this is a synonym of [Alpha-aminoadipic aciduria HP:0410309](https://hpo.jax.org/app/browse/term/HP:0410309). See also [adipic acid](https://www.ebi.ac.uk/ols/ontologies/chebi/terms?iri=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FCHEBI_30832).
Can you provide more context?

**Comment by pnrobinson** [comment]
@MickeySegal 
I think these are actually the same compound. This is what GPT has to say

Adipic acid and alpha adipic acid refer to the same chemical compound. The term "adipic acid" is the standard name for this substance, and it is widely recognized and used in chemical nomenclature. Here's a detailed explanation:

The term "alpha adipic acid" is not commonly used in chemistry and does not refer to a different compound. It seems to be a misnomer or a confusion with other nomenclature conventions.
Possible Confusion: In organic chemistry, the prefix "alpha" (α) is sometimes used to denote the position of a substituent relative to a functional group, typically referring to the first carbon atom adjacent to the functional group. However, in the case of adipic acid, such a designation is unnecessary and not used.

**Comment by MickeySegal** [comment]
This is a classic example of ChatGPT misunderstanding the question and giving an answer that sounds authoritative but is completely tangential to the query.  

Wikipedia is more helpful here, showing the structures:
https://en.wikipedia.org/wiki/Adipic_acid
https://en.wikipedia.org/wiki/%CE%91-Aminoadipic_acid

As expected from the name, alpha amino adipic acid has an amino group added to adipic acid.

This is why people striving to do medical diagnosis accurately should depend on you and me and the information we curate and not on ChatGPT.

**Comment by pnrobinson** [comment]
@MickeySegal 
indeed!

Adding new term: 
- ID: HP:6001114
- label: Elevated urinary adipic acid
- parents: [Dicarboxylic aciduria (HP:0003215)](https://hpo.jax.org/app/browse/term/HP:0003215)
- pmid: 
- def: The amount of adipic acid in the urine, normalized for urine concentration, is above the upper limit of normal. Adipic acid is an alpha,omega-dicarboxylic acid that is the 1,4-dicarboxy derivative of butane.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8599
- synonyms:
